ninetofiver/models.py

```diff
@@ -1134,4 +1137,13 @@ def get_validation_args(self):
             """Get a dict used for validation based on this instance."""
             return merge_dicts(super().get_validation_args(), {
                 'performance': getattr((self, 'performance', None)),
-            })
\ No newline at end of file
+            })
+
+
+###########################################
```

Comment:
No need for this code here. We don't store duplicated redmine data in our database.